style.css

```diff
@@ -45,7 +45,7 @@
         }
  
         button {
-            background-color: #05c46b;
+            background-color: #05C46B;
```

Comment:
You are suggested to edit the background-color. Please white every alphabets in lower case. This is the best practice.